raiden/ui/cli.py

```diff
@@ -68,6 +68,13 @@
         type=str,
     ),
     click.option(
+        '--rpccorsdomain',
+        help='Comma separated list of domains to accept cross origin requests. \n'
+        '(localhost enabled by default)',
+        default=None,
```

Comment:
Why not put the localhost as the `default` here and if people want to provide specific ones they would have to also provide the local host? Asking because right now as you have it, even if you did not want to add the CORS for local host you can't/